js/libs/form.js

```diff
@@ -222,7 +222,9 @@ Fliplet.Widget.instance('form-builder', function(data) {
 
         Fliplet.Navigator.onOffline(function() {
           $vm.isOffline = true;
-          $vm.isOfflineMessage = data.dataStore && data.dataStore.indexOf('editDataSource') > -1 ? 'The data can only be updated when connected to the internet.' : 'This form can only be submitted when connected to the internet.';
+          $vm.isOfflineMessage = data.dataStore && data.dataStore.indexOf('editDataSource') > -1 ?
+            'The data can only be updated when connected to the internet.' :
+            'This form can only be submitted when connected to the internet.';
```

Comment:
@squallstar My linter doesn't agree with your suggested way. So I kept the linter recommendation!